CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 # Version 2.x
 
+# 2.0.0 (unreleased)
+
+* Removed `PropertyTypeArray`, which is superseeded by `PropertyTypeIterable`.
+* Removed the deprecated `PropertyTypeIterable::getCollectionClass`. Use `PropertyTypeIterable::getTraversableClass`
+* Removed the deprecated `PropertyTypeIterable::isCollection`. Use `PropertyTypeIterable::isTraversable`
```

Comment:
```suggestion
* Removed the deprecated `PropertyTypeIterable::isCollection`. Use `PropertyTypeIterable::isTraversable`
* `JMSTypeParser::getTraversableClass` returns `Traversable::class` instead of `Doctrine\Common\Collections\Collection` for general traversable properties.
```